python-backend/tests/mines/resources/test_mine_resource.py

```diff
@@ -301,6 +316,17 @@ def test_put_mine_name(test_client, auth_headers):
     assert put_resp.status_code == 200
 
 
+def test_put_redundant_mine_name(test_client, auth_headers):
+    # this relies on  test test_post_mine_name_and_coord
+    test_tenure_data = {
+        "name": "test_create_mine",
+        "note": ""
```

Comment:
Is the `note` field required here? I don't think it should be. If it isn't, we should probably remove it to keep the test as specific as possible.